astacus/coordinator/manifest.py

```diff
@@ -5,10 +5,26 @@
 from astacus.common import asyncstorage, ipc
 from starlette.concurrency import run_in_threadpool
 
+import msgspec
+
 
 async def download_backup_manifest(json_storage: asyncstorage.AsyncJsonStorage, backup_name: str) -> ipc.BackupManifest:
-    d = await json_storage.download_json(backup_name)
-    manifest = await run_in_threadpool(ipc.BackupManifest.parse_obj, d)
+    def download_manifest() -> ipc.BackupManifest:
+        with json_storage.storage.open_json_bytes(backup_name) as manifest_content:
+            return msgspec.json.decode(manifest_content, type=ipc.BackupManifest)
```

Comment:
Might be time to retire `AsyncJsonStorage` IMO if we are bypassing the abstraction like this.

Reply:
Yes, but in a separate PR.